aerospike/README.md

```diff
@@ -11,19 +11,32 @@ Get metrics from Aerospike Database in real time to:
 
 ### Installation
 
-The Aerospike check is included in the [Datadog Agent][1] package.
+The Aerospike check is included in the Datadog Agent package.
 No additional installation is needed on your server.
 
 ### Configuration
 
+Two dashes
```

Comment:
What is two dashes?

Reply:
Just running tests, this one isn't ready for review yet